model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -318,7 +318,15 @@ message StandardPTransforms {
 
 message StandardSideInputTypes {
   enum Enum {
+    // Represents a view over a PCollection<V>.
+    //
+    // The SDK is limited to perform state get requests using the StateKey.IterableSideInput.
     ITERABLE = 0 [(beam_urn) = "beam:side_input:iterable:v1"];
+
+    // Represents a view over a PCollection<KV<K, V>>.
+    //
+    // The SDK is able to perform state get requests with the StateKey.IterableSideInput and
```

Comment:
In the same vein as the comments above, I think these lines would be less ambiguous like so:

"StateGetRequests performed on this side input must use StateKey.IterableSideInput or StateKey.MultimapSideInput."

Like my other comments, the important change is that StateGetRequests are explicitly mentioned as _using_ the StateKey and being performed _on_ the side input represented by this URN.